### PR TITLE
More functional tests on new functions

### PR DESCRIPTION
**Pull request by philippethiran**

### CODE REVIEW

File: sum/another_sum.py [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+def add_serie(a, b):
+    y = 0
+    for x in range(a, b + 1):
+        y = y + x
+    return y
+
+def add_even_serie(a, b):
+    # this function add only the even numbers between a and b
+    y = 0
+    if (a % 2) != 0 :  # a is not even
```
Comment (philippeunamur) [inline sum/another_sum.py]: PEP8: no withespace before :

File: sum/another_sum.py [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
-    return a + 10
\ No newline at end of file
+    return a + 10
+
+
+def add_serie(a, b):
+    y = 0
+    for x in range(a, b + 1):
+        y = y + x
+    return y
```
Comment (philippeunamur) [inline sum/another_sum.py:13]: 2 blank lines are expected here.

File: sum/another_sum.py [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
     return a + b
 
+
 def add_10(a):
-    return a + 10
\ No newline at end of file
+    return a + 10
+
+
+def add_serie(a, b):
```
Comment (philippeunamur) [inline sum/another_sum.py]: add_serie -> add_series

File: sum/another_sum.py [hunk 4]
```diff
@@ -1,5 +1,22 @@
...
+    y = 0
+    for x in range(a, b + 1):
+        y = y + x
+    return y
+
+def add_even_serie(a, b):
```
Comment (philippeunamur) [inline sum/another_sum.py]: add_even_serie -> add_even_series

**Review by philippeunamur — CHANGES_REQUESTED** [review]

File: sum/another_sum.py (hunk 2 above)
Comment (philippethiran) [reply sum/another_sum.py:13]: Corrected

**Review by philippethiran — COMMENTED** [review]
Everything corrected

**Pull request merged**